tests/e2e/pageObjects/types.ts

```diff
@@ -0,0 +1,18 @@
+export type StandardWalletType = '3-Key Vault' | '2-Key Vault' | 'Standard HD P2SH';
```

Comment:
(?) What do you think to change the file name from `types.js` to something more descriptive i.e. `walletTypes.js`?

Reply:
Not really. The file includes 3 other types, it's not only about the wallet types.

Apart from that, there's not a convention in TS to have a file per single type/interface. The TypeScript team uses as well a file called `types.ts`: https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts